pipeline-runner/R/handle_data.R

```diff
@@ -250,6 +250,19 @@ upload_matrix_to_s3 <- function(pipeline_config, experiment_id, data) {
   return(object_key)
 }
 
+upload_debug_folder_to_s3 <- function(debug_subdir, pipeline_config) {
+  fnames <- list.files(file.path("/debug", debug_subdir))
```

Comment:
this `/debug` folder name is used all over the code. How about you define a config file (like we have in the python and in the r worker) and in then you move it there in the config folder as a config variable and then import and read it in various places in the code?
Btw, we will need to do this for multi-account deployments for aws resource names and aws region and so on

Reply:
I actually see now that you already have config defined `init.R`. So I would add the debug folder in there too

Reply:
yes this is something we are aware of and should be addressed. The same is true of `work.R` in the worker. They are both largely untested